packages/datadog-instrumentations/src/opensearch.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const { addHook } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+const { createWrapRequest } = require('./elasticsearch')
```

Comment:
At this way, if `@opensearch-project/opensearch` is used, we will addHooks also to `elasticsearch`. If we extract this method to other file used from `opensearch.js` and `elasticsearch.js` we will prevent it.

Reply:
Actually, given the current design, it would be a tad difficult to extract this and it shouldn't matter anyway if additional hooks are registered, especially if it's with the same function since this will have no overhead.